extensions/ql-vscode/src/extension.ts

```diff
@@ -493,6 +494,29 @@ async function activateWithInstalledDistribution(
     }
   }
 
+  async function previewQueryHelp(
+    selectedQuery: Uri
+  ): Promise<void> {
+    const path = selectedQuery ? selectedQuery.path : window.activeTextEditor!.document.uri.fsPath;
+    if(path) {
+      // Create temporary directory
+      const tmpDir = tmp.dirSync();
+      // Get library name and query file name in the format 'libname/queryname.md
+      const mdFilePath = path.split('/').slice(-2).join('/').replace('.qhelp', '.md');
+      const absolutePathToMd = `${tmpDir.name}${mdFilePath}`;
+      const uri = Uri.file(absolutePathToMd);
+      try {
+        await cliServer.generateQueryHelp(path , 'markdown', absolutePathToMd);
+        await commands.executeCommand('markdown.showPreviewToSide', uri);
+      } catch (err) {
+        const errorMessage =  err.message.includes('Generating qhelp in markdown') ? (
+          `Could not generate markdown from ${path}: Bad formatting in .qhelp file.`
+        ) : `Could not open a preview of the generated file (${absolutePathToMd}).`;
+        void helpers.showAndLogErrorMessage(errorMessage);
```

Comment:
I think this will suppress the error message from the CLI, which is hopefully useful. Could we include `err` in the message we construct here?